rocketpy/prints/flight_prints.py

```diff
@@ -4,5 +4,423 @@
 
 
 class _FlightPrints:
-    def __init__(self) -> None:
-        pass
+    """Class that holds prints methods for Flight class.
+
+    Attributes
+    ----------
+    _FlightPrints.flight : Flight
+        Flight object that will be used for the prints.
+
+    """
+
+    def __init__(
+        self,
+        flight,
+    ):
+        """Initializes _FlightPrints class
+
+        Parameters
+        ----------
+        flight: Flight
+            Instance of the Flight class.
+
+        Returns
+        -------
+        None
+        """
+        self.flight = flight
+        return None
+
+    def printInitialConditionsData(self):
```

Comment:
Since this is already a print module, I think there's no need to write "print" in the beggining of each method.
Also, this is a new file and will be deployed at v1.0.0, so I would accelerate the things here and write everything in snake_case. 

The following suggestion seems better to me, and for you @MateusStano ? 
```suggestion
    def initial_conditions(self):
```

Reply:
Agreed! This makes more sense

Will commit the changes right now